roles/pool/README.md

```diff
@@ -18,47 +20,32 @@ The most typical high level configuration is:
 ```
 
 ## Setup
+
 ### Configuration File
-The `pool-config-example.toml` is a configuration example which can be copy/paste into `/conf` directory by the party that is running the SV2 Pool (most
-typically the pool service provider) to address the most preferred customization.
+
+`pool-config-hosted-tp-example.toml` and `pool-config-local-tp-example.toml` are examples of configuration files.
 
 The configuration file contains the following information:
 
-1. The SRI Pool information which includes the SRI Pool authority public key 
-   (`authority_pubkey`), the SRI Pool authority secret key (`authority_secret_key`), along with its certificate validity (`cert_validity_sec`). In addition to this, it contains the address which it will use to listen to new connection from downstream roles (`listen_address`) and the list of uncompressed pubkeys for coinbase payout (`coinbase_outputs`).
+1. The SRI Pool information which includes the SRI Pool authority public key
+   (`authority_public_key`), the SRI Pool authority secret key (`authority_secret_key`), along with its certificate validity (`cert_validity_sec`). In addition to this, it contains the address which it will use to listen to new connection from downstream roles (`listen_address`),it has the list of uncompressed pubkeys for coinbase payout (`coinbase_outputs`) and a string that serves as coinbase tx known as (`pool_signature`).
```

Comment:
```suggestion
1. The SRI Pool information which includes the SRI Pool authority public key
   (`authority_public_key`), the SRI Pool authority secret key (`authority_secret_key`).
 2. The address which it will use to listen to new connection from downstream roles (`listen_address`)
 3. The list of uncompressed pubkeys for coinbase payout (`coinbase_outputs`)
 4. A string that serves as signature on the coinbase tx (`pool_signature`).
```